ONSdigital/dp-dd-api-stub
# New api

##What

Set of new JSONs based on the idea of the new API structure.

### How to review

- Make sure the JSON files in the stub represent the data the backend wants to provide and the front-end wants to consume
- Make sure the changes to the handlers are not to erroneous 
- Run `make debug` and make sure you can access the resources you require

### Who can review

@NeilMadden @fullstackforger
